configure.php

```diff
@@ -653,22 +653,20 @@ function install_plugin( array $plugin_data, bool $prompt, &$installed_plugins )
 		run( 'ln -s mu-plugins/drop-ins/object-cache.php object-cache.php' );
 	}
 
-	write( 'Scaffolding out vip-config...' );
-
-	run( 'mkdir -p vip-config && touch vip-config/.gitkeep' );
-
 	write( 'Scaffolding out VIP directories...' );
 
 	run( 'mkdir -p images && touch images/.gitkeep' );
 	run( 'mkdir -p languages && touch languages/.gitkeep' );
 	run( 'mkdir -p private && touch private/.gitkeep' );
+	run( 'touch vip-config/.gitkeep' );
 
 	echo "Done!\n\n";
 } elseif ( 'pantheon' === $hosting_provider ) {
 	write( 'Deleting VIP-specific GitHub Action workflows...' );
 
 	delete_files(
 		[
+			'vip-config',
```

Comment:
when/where is this being created that it is necessary to delete this at this point?

Reply:
@jakewrfoster I guess as part of the repo we have `vip-config/vip-config.php` that needs to be removed?